src/Parser.js

```diff
@@ -764,52 +706,60 @@ export default class Parser {
     }
 
     /**
-     * If `optional` is false or absent, this parses an ordinary group,
-     * which is either a single nucleus (like "x") or an expression
-     * in braces (like "{x+y}") or an implicit group, a group that starts
-     * at the current position, and ends right before a higher explicit
+     * Parses an argument.
+     */
+    parseArgumentGroup(optional: boolean, mode?: Mode): ?AnyParseNode {
```

Comment:
Could the result be typed as `?ParseNode<"ordgroup">` or does that cause other issues with the flow types?

Reply:
@kevinbarabash Yes, I'll update the PR.